README.md

```diff
@@ -1,3 +1,110 @@
-# orders
-DevOps homework project for CSCI-GA.2820-001 DevOps and Agile Methodologies Fall 2021.
+# Orders
+NYU DevOps lab on Test Driven Development for CSCI-GA.2820-001 DevOps and Agile Methodologies Fall 2021.
```

Comment:
we can revert back to the previous description here : DevOps homework project for CSCI-GA.2820-001 DevOps and Agile Methodologies Fall 2021.